tests/test_job_manager.py

```diff
@@ -2,12 +2,12 @@
 import pytest
 from datetime import datetime
 from metriq_gym.job_manager import JobManager, MetriqGymJob
-from tests.test_schema_validator import TEST_BENCHMARK_NAME, TestJobType
+from tests.test_schema_validator import TEST_BENCHMARK_NAME, FakeJobType
```

Comment:
While I know the rename thing is a `pytest` quirk, do we want to also rename `TEST_BENCHMARK_NAME` to `FAKE_BENCHMARK_NAME` for consistency?